src/app/footer/footer.component.html

```diff
@@ -1 +1 @@
-<div id="footer_container" class="footer width100 fl">Copyright 2016 Red Hat</div>
+<div id="footer_container" class="footer width100 fl dn">Copyright 2016 Red Hat</div>
```

Comment:
There is no design or workflow for footer when we have Quick Add At Bottom, So I just hide it